src/test/acceptance/features/address-lookup.feature

```diff
@@ -37,3 +37,10 @@ Feature: Select address
     Given I have entered my details up to the select address page
     When I click the address not listed link
     Then I am shown the manual address page
+
+  # TODO DW HTBHF-2037 include test once address lookup is enabled
+  @ignore
+  Scenario: Changing details for a selected address takes me to the select address page
+    Given I have entered my details up to the check details page and selected an address
```

Comment:
The term `selected an address` isn't clear enough, we should probably mention postcode lookup here otherwise its ambiguous.

Reply:
There's only one way to get to select address, so can't see how it's ambiguous.